### PR TITLE
Changed package names. Removed untested(unused) pieces of code.

### PR DESCRIPTION
**Pull request by LasTshaMAN**
Renamed packages.

Removed pieces of code that were there just for the sake of being there and didn't bring any functional payload. (If you want to return them back, write tests that meaningfully use them).

Changed hasCode and equals. Looks like they weren't auto generated by IDEA, for some reason they were different from what IDEA generated for me.

### CODE REVIEW

File: src/main/java/nc/students/ayaz/Video.java [hunk 1]
```diff
@@ -44,4 +24,19 @@ void addComment(Comment comment) {
     void deleteComment(Comment comment) {
         comments.remove(comment);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```
Comment (ghost) [inline src/main/java/nc/students/ayaz/Video.java:33]: &#x1F34E; **Issue**
Why cant we remove this row? If we have same objects then equals bellow will return true.

File: src/main/java/nc/students/ayaz/Comment.java [hunk 2]
```diff
@@ -7,20 +7,20 @@
...
         this.text = text;
-        this.date = Calendar.getInstance().getTime();
     }
 
-    private String getAuthor() {
-        return author;
-    }
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```
Comment (ghost) [inline src/main/java/nc/students/ayaz/Comment.java:18]: &#x1F34E; **Issue**
Why cant we remove this row? If we have same objects then equals bellow will return true.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/nc/students/ayaz/Video.java (hunk 1 above)
Comment (LasTshaMAN) [reply src/main/java/nc/students/ayaz/Video.java:33]: It is auto generated code. (by IDEA)
That means they know what they are doing ))
But the actual reason, I think, is performance. Its much cheaper to compare references than compare full objects.

File: src/main/java/nc/students/ayaz/Comment.java (hunk 2 above)
Comment (LasTshaMAN) [reply src/main/java/nc/students/ayaz/Comment.java:18]: Same answer here.

File: src/main/java/nc/students/ayaz/Video.java (hunk 1 above)
Comment (ghost) [reply src/main/java/nc/students/ayaz/Video.java:33]: &#x1F34F; **Revoked**

File: src/main/java/nc/students/ayaz/Comment.java (hunk 2 above)
Comment (ghost) [reply src/main/java/nc/students/ayaz/Comment.java:18]: &#x1F34F; **Revoked**

**Review by ghost — APPROVED** [review]

**Pull request merged**